packages/sdk/src/streamStateView_Space.ts

```diff
@@ -27,21 +27,13 @@ export type ParsedChannelProperties = {
 export class StreamStateView_Space extends StreamStateView_AbstractContent {
     readonly streamId: string
     readonly spaceChannelsMetadata = new Map<string, ParsedChannelProperties>()
-    private _spaceImage: ChunkedMedia | undefined
+    private spaceImage: ChunkedMedia | EncryptedData | undefined
```

Comment:
I really don’t think we need a union here. How about two variables? One for the encrypted payload, one for the decrypted one.

Reply:
sure